contrib/kubespray/roles/port/defaults/main.yml

```diff
@@ -0,0 +1 @@
+host_daemon_port_range: "40000 60000"
```

Comment:
Current job ports range also needs to exclue low range ports, such as 80, 86, etc.
So, how about change it to 0 19999, so the available job ports range is 20000 to 65535?

Pls also check PAI other services ports, that them should be within  0 19999

Reply:
Sure, will change this value.
Checked pai services. 
hived webservice-port is 30096 by default: https://github.com/microsoft/pai/blob/bdf7f49902ebf613ff90c79ad19ae21c8cd358d9/src/hivedscheduler/config/hivedscheduler.yaml#L21

Other services are using ports within 0,19999

Reply:
If so, could u pls also change hived to be 10096?

Reply:
Fixed @abuccts , please help to check

Reply:
you do not need to check ports for services which only run on master nodes

Reply:
Refer to https://kubernetes.io/docs/concepts/services-networking/service/#nodeport, k8s service node port range is: ` 30000-32767. This change will break CI.

If it only run on master node, will revert this change